invenio_app_rdm/config.py

```diff
@@ -702,6 +700,29 @@ def files_rest_permission_factory(obj, action):
 ]
 """Template names for detail view sidebar components"""
 
+APP_RDM_FILES_INTEGRITY_REPORT_TEMPLATE = (
```

Comment:
I agree with the naming convention but just for clarification this was not needed for the config to be picked up as the invenio-app-rdm config is defined as `invenio_config.module`in setup.cfg.